src/main/java/com/atlassian/shele/shelePlugin/customfield/SimpleCustomField.java

```diff
@@ -3,23 +3,18 @@
 import com.atlassian.jira.issue.customfields.impl.GenericTextCFType;
 import com.atlassian.jira.issue.customfields.manager.GenericConfigManager;
 import com.atlassian.jira.issue.customfields.persistence.CustomFieldValuePersister;
-import com.atlassian.jira.security.JiraAuthenticationContext;
 import com.atlassian.plugin.spring.scanner.annotation.imports.ComponentImport;
 
 /**
- * SimpleCustomField module access for change item only
- * amin
+ * SimpleCustomField module for examining how create a customfield
```

Comment:
is this the correct description or the field should be available for editing only for admins and shown to everybody?
please also rename the field class to reflect its purpose